cmd/commands/server/command_server.go

```diff
@@ -93,15 +95,14 @@ func (cmd *Command) Start() (err error) {
 		case openvpn.ExitingState:
 			log.Info("Open vpn service exiting")
 			close(stopPinger)
+			cmd.mysteriumClient.UnregisterProposal(providerID.Address, signer)
```

Comment:
Don't swallow possibly returned error - at least log it.

Reply:
Also this unregistration+logging is duplicated. I would like same cleanup function here e.g. `cmd.Kill()`